src/memkind_malloc.c

```diff
@@ -33,19 +34,22 @@ void *memkind_calloc_wrapper(size_t size) {
         size = jemk_malloc_usable_size(ptr);
         update_memkind_malloc_stat_alloc(size);
     }
+    printf("Calloc:\t%p\n",ptr);
```

Comment:
I think changes in memkind_malloc.c file are unrelated to  "Memkind and Jemalloc static link"